sdk/identity/Azure.Identity/src/ManagedIdentityCredential.cs

```diff
@@ -14,23 +14,28 @@ namespace Azure.Identity
     public class ManagedIdentityCredential : TokenCredential
     {
         private string _clientId;
-        private IdentityClient _client;
+        private ManagedIdentityClient _client;
+        private SemaphoreSlim _initLock = new SemaphoreSlim(1, 1);
+
+
+        private readonly Uri ImdsEndptoint = new Uri("http://169.254.169.254/metadata/identity/oauth2/token");
```

Comment:
These two constants are probably not needed now that they're in `ManagedIdentityClient`

Reply:
removed